packages/bar/src/Bar.tsx

```diff
@@ -420,6 +436,21 @@ const InnerBar = <RawDatum extends BarDatum>({
 
                 return layerById?.[layer] ?? null
             })}
+            {barTotals.map(barTotal => (
```

Comment:
Ah, sorry, I should have been more explicit about using a flag to enable the feature, I think it should come on top of the layers feature (with the layer present by default), layers can be useful not only to control if something is present or not, but also to control the order of the layers, which can be important in certain situations.

Reply:
Got it, so the prop should control the layer array. Just to check if I got that right, should it be enabled by default?

Reply:
Those would be independent, for example, this is how it's done for the labels of the `Chord` diagram:

```
if (layers.includes('labels') && enableLabel) {
```

And `labels` is part of the layers by default.

Reply:
Got it! So it is implemented using layers, but controlled mainly through prop.